### PR TITLE
Disable remote users preview images generation

### PR DESCRIPTION
**Pull request by joachimesque**
Right now, preview images are automatically generated for external accounts. IMO it's useless: it takes up disk space, and uses computing power.

This PR contains:

- a fix that adds a check on the `user.local` bool (with test)
- a management command that deletes preview images for non local users (with test and shell completion)

### CODE REVIEW

File: bookwyrm/management/commands/remove_remote_user_preview_images.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            "   | Hello! I will be removing preview images from remote users."
+        )
+        self.stdout.write(
+            "🧑‍🚒 ⎨ This might take quite long if your instance has a lot of remote users."
+        )
+        self.stdout.write("   | ✧ Thank you for your patience ✧")
+
+        users = models.User.objects.filter(local=False).exclude(
+            Q(preview_image="") | Q(preview_image=None)
+        )
```
Comment (joachimesque) [inline bookwyrm/management/commands/remove_remote_user_preview_images.py:27]: This query is quite power-intensive. As this command is destined to be ran a couple of times at most in the life of the instance, I don't know if it's worth being optimized

Comment (mouse-reeve) [reply bookwyrm/management/commands/remove_remote_user_preview_images.py:27]: I think it's fine as is, given, like you said, that it's not meant ot be in regular use

File: bookwyrm/preview_images.py [hunk 2]
```diff
@@ -453,12 +453,15 @@ def generate_edition_preview_image_task(book_id):
...
 def generate_user_preview_image_task(user_id):
-    """generate preview_image for a book"""
+    """generate preview_image for a user"""
     if not settings.ENABLE_PREVIEW_IMAGES:
         return
 
     user = models.User.objects.get(id=user_id)
 
+    if not user.local:
+        return
```
[the remark this answers is not in the document]
Comment (joachimesque) [reply bookwyrm/preview_images.py:463]: Added the check in [a44f427](https://github.com/bookwyrm-social/bookwyrm/pull/2522/commits/a44f427d84641c67d0bd1654592c8ed891a2cd7a). I don't know how best to test it

[the remark this answers is not in the document]
Comment (mouse-reeve) [reply bookwyrm/preview_images.py:482]: I think it would be more trouble than it's worth to test that line (at minimum within the scope of this PR)

**Review by mouse-reeve — APPROVED** [review]
